.travis.yml

```diff
@@ -1,26 +1,55 @@
 language: rust
-sudo: true
-rust:
-  - stable
-  - beta
-  - nightly
-  - 1.27.0
-  - 1.26.0
-  - 1.25.0
-  - 1.24.0
-  - 1.23.0
-  - 1.22.0
-os:
-  - linux
-  - osx
+cache: cargo
```

Comment:
why is this needed? for me I prefer to download latest cargo-make each time (its another test in a way)

Reply:
Deleted.